nuxeo/templates/ingress.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       {{- with .Values.ingress.hostname }}
       host: {{ toYaml . }}
       {{- end }}
-  {{- with .Values.ingress.tls }}
-  tls: {{- toYaml . | nindent 4 }}
+  {{- if .Values.ingress.tls }}
+  tls: {{- tpl (toYaml .Values.ingress.tls) . | nindent 4 }}
```

Comment:
For my curious, what kind of cases is it useful?

Reply:
In the wrapper chart values for the GCP marketplace app: https://github.com/nuxeo/nuxeo-gcp-marketplace/pull/1/commits/d6505505e7295d3f874b5d7f880f284e4a28c00a#diff-bfcbb283c09032173229cbcbfee59f7a63db27e1b70880c79889c76d96729bb6R10, to have a dynamic TLS secret name (depending in the release name) and its hostname matching the ingress hostname (defined [here](https://github.com/nuxeo/nuxeo-gcp-marketplace/pull/1/commits/d6505505e7295d3f874b5d7f880f284e4a28c00a#diff-bfcbb283c09032173229cbcbfee59f7a63db27e1b70880c79889c76d96729bb6R8)).